src/runtime_src/core/pcie/tools/xbmgmt/cmd_config.cpp

```diff
@@ -155,10 +160,22 @@ static void showDevConf(std::shared_ptr<pcidev::pci_device>& dev)
     if (!errmsg.empty()) {
         std::cout << "can't read security level from " << dev->sysfs_name <<
             " : " << errmsg << std::endl;
-        return;
+    } else {
+        std::cout << dev->sysfs_name << ":" << std::endl;
```

Comment:
std::endl should be used only if you wish to flush the stream.  Use "\n" for all but last std::cout.